src/gui/tray/activitydata.h

```diff
@@ -76,10 +76,16 @@ class Activity
         QString path;    // Required (for files only)
         QUrl link;    // Optional (files only)
     };
+    
+    struct TalkNotificationData {
+        QString conversationToken{};
```

Comment:
@camilasan `QString` is constructed empty by default, so, this `{}` is really redundant